components/Card.test.js

```diff
@@ -1,4 +1,4 @@
-import { render } from '@testing-library/react'
+import { render, act } from '@testing-library/react'
```

Comment:
We can remove `act` as it's not being used :)

Reply:
Good thing you noticed. I had added act to the test to fix the warning but it was removed on merge.

Reply:
I see act being used on line 25